### ISSUE TITLE
Tutorial on real research workflow (long-term simulations with large data)

### ISSUE THREAD

**Issue by JiaweiZhuang** [geoschem/cloud_GC#4]
After #2 is done, we will be able to run any types of GEOS-Chem simulations over any periods. We can go beyond proof-of-concept runs and do serious research projects.

Preliminary workflow design:
1. Use small instances (r4.large) for model configuration and testing.
2. Switch to more powerful instances (>c5.4xlarge) to perform long-term simulations. Use spot instances to bring down the computing cost.
3. For large output data, move to S3 to bring down the storage cost (also see #3).
4. Use small instances (c5.large) for data analysis.

The working directory should probably be in a standalone EBS volume, so can it can be quickly shared between instances and will not be affected by spot instance termination. The root volume should only have software libraries and model source code.

**Comment by JiaweiZhuang** [comment]
Example workflow now posted:

http://cloud-gc.readthedocs.io/en/latest/chapter02_beginner-tutorial/research-workflow.html

**Comment by JiaweiZhuang** [comment]
Re-open this issue since we still need to investigate what's the "best workflow" on the cloud. This could depend on specific research projects, including as long the simulation takes, how many simulations needs to run, etc. But it would be nice to have a "sample workflow" for a serious project.
